purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -992,8 +992,8 @@ class Purchases internal constructor(
             appInBackground,
             { offeringsJSON ->
                 try {
-                    val skus = extractSkus(offeringsJSON)
-                    if (skus.isEmpty()) {
+                    val productGroupIdentifiers = extractProductGroupIdentifiers(offeringsJSON)
```

Comment:
i don't think this will work for BC4

Reply:
Looks like it doesn't need to anymore, right? https://revenuecat.slack.com/archives/C03M70X8DGF/p1666956760783019

Reply:
true, if we go that route a lot can be simplified!